src/App.tsx

```diff
@@ -1,10 +1,55 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useState,
+  useCallback,
+  useEffect,
+} from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { NewTodo } from './components/NewTodo';
 
-const USER_ID = 0;
+const USER_ID = 10358;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState<boolean>(false);
+  const [filterBy, setFilterBy] = useState<string>('All');
```

Comment:
Consider using a specific type for 'filterBy' state, e.g. type Filter = 'All' | 'Active' | 'Completed';